thunder-service/controller/routing.go

```diff
@@ -44,4 +44,7 @@ func (ctrl *ThunderController) setupProtectedRoutes(router *mux.Router) {
 	router.HandleFunc("/oracle-cloud/loadbalancers/{ids}", ctrl.GetOciUnusedLoadbalancers).Methods("GET")
 	router.HandleFunc("/oracle-cloud/instancesidle/{ids}", ctrl.GetOciComputeInstancesIdle).Methods("GET")
 	router.HandleFunc("/oracle-cloud/blockstorage/{ids}", ctrl.GetOciBlockStorageRightsizing).Methods("GET")
+	router.HandleFunc("/oracle-cloud/unusedstorage/{ids}", ctrl.GetOciUnusedStorage).Methods("GET")
```

Comment:
Would you mind to change this in `unused-storage`?
And the same for the others api, split words with `-` please :)

Reply:
Changes applied